model/Core/Classes/Requirement.md

```diff
@@ -0,0 +1,39 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Requirement
+
+## Summary
+
+A distinct unit representing a requirement as it is used by systems, software and hardware engineering.
+
+## Description
+
+A requirement element is a distinct article or unit defining an expectation, need, behaviour, design intent etc. of an item that already exists or is to be created based on this requirement.
+
+## Metadata
+
+- name: Requirement
+- SubclassOf: Element
+
+## Properties
+
+- requirementUID
+  - type: xsd:anyURI
+  - minCount: 0
+  - maxCount: 1
+- devLifeCycleStage
+  - type: LifecycleScopeType
+  - minCount: 0
+  - maxCount: *
+- requirementText
```

Comment:
Is it possible to reuse `/Core/description` here?
It is available to all Elements.
https://spdx.github.io/spdx-spec/v3.0.1/model/Core/Properties/description/

Then we can add something similar to this to the `Requirement`'s description above:

> Statement describing the actual need can be put inside `description`.
>
> Rational or additional information can be put inside `comment`.

Reply:
We can just merge this and amend it later.
I just ask because of curiosity.